<java.util.concurrent.FutureTask:setException,parmtr,1,<>> = <<java.util.concurrent.FutureTask:setException,argmnt,1,<null>>,E,E>;
<java.util.concurrent.FutureTask:setException,parmtr,2,<>> = <<java.util.concurrent.FutureTask:setException,argmnt,2,<null>>,E,E>;
<java.util.concurrent.FutureTask:setException,local1,2,<>> = <<java.util.concurrent.FutureTask:setException,local1,2,<null>>,E,E>;
<java.util.concurrent.FutureTask:setException,local1,22,<>> = <<java.util.concurrent.FutureTask:setException,local1,22,<null>>,E,E>;
